FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -35,9 +36,139 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                            <locus from="71r" to="78v"/>
+                                        </item> 
+                                        <item xml:id="q11" n="11">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="79r" to="86v"/>
+                                        </item> 
+                                        <item xml:id="q12" n="12">
+                                            <dim unit="leaf">9</dim>
+                                            <locus from="87r" to="95v"/>
+                                            s.l. 6, no stub
+                                        </item> 
+                                        <item xml:id="q13" n="13">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="96r" to="103v"/>
+                                        </item> 
+                                        <item xml:id="q14" n="14">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="104r" to="111v"/>
+                                        </item> 
+                                        <item xml:id="q15" n="15">
+                                            <dim unit="leaf">2</dim>
+                                            <locus from="112r" to="113v"/>
+                                        </item> 
+                                    </list>
+                                    <item xml:id="q16" n="16">
+                                        <dim unit="leaf">6</dim>
+                                        <locus from="114r" to="119v"/>
+                                        s.l. 4, stub after 5,
+                                        s.l. 5, stub after 5,
+                                        s.l. 7, no stub,
+                                        s.l. 8, no stub
+                                        <note>Contrary to Marrassini, who considered <locus target="#118"/> and <locus target="#119"/> separate from 
+                                            <ref target="#q16"/>. The text in this quire seemingly show no breaks.</note>       
+                                    </item>
+                                    <item xml:id="q17" n="17">
+                                        <dim unit="leaf">3</dim>
+                                        <locus from="120r" to="122v"/>
+                                        s.l. 1, stub after 3
+                                    </item>
+                                    <item xml:id="q18" n="18">
+                                        <dim unit="leaf">9</dim>
+                                        <locus from="123r" to="131v"/>
+                                        s.l. 10, stub before 4
+                                        s.l. 11, stub before 4
```

Comment:
In this quire, there can be no leaves 10, 11 and 12, since it has only 9 leaves all in all.